### ISSUE TITLE
FRENCH MANUAL 1.10

### ISSUE THREAD

**Issue by ronron84** [FrSkyRC/ETHOS-Feedback-Community#850]
[X20 et Ethos1.010a.docx](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/7040007/X20.et.Ethos1.010a.docx)
Here is the french manual 1.0.10

**Comment by Adelaxie** [comment]
Can I share it to the manual link under code section？
thanks for your translation

**Comment by ronron84** [comment]
yes sure, if this document might help some modellers to go further it's a good thing. Thanks for your work !

**Comment by Adelaxie** [comment]
great！done @ronron84
